### DISCUSSION TITLE
`Query Key Factory Package` link is redirecting in the docs

### DISCUSSION THREAD

**Discussion by martypowell** [TanStack/query#6286] [General]
While visiting https://tanstack.com/query/latest/docs/react/guides/query-keys#further-reading I noticed the [Query Key Factory Package](https://tanstack.com/query/latest/docs/react/community/lukemorales-query-key-factory) link is redirecting to https://tanstack.com/query/latest/docs/react/overview.

**Comment by TkDodo** [comment]
thanks. it should probably either link here:
- https://tanstack.com/query/latest/docs/react/community/community-projects#query-key-factory

or just generally to the community resources.

want to submit a PR ?

**Reply by martypowell** [reply]
@TkDodo yep I can do that.

**Reply by martypowell** [reply]
@TkDodo https://github.com/TanStack/query/pull/6316
